### ISSUE TITLE
Typo on "vault operator init" command. 

### ISSUE THREAD

**Issue by ko5tas** [hashicorp/vault#11727]
**Describe the bug**
Small typo on the output of the "vault operator init" command. 
It prints 
"Without at least 3 **key** to reconstruct the master key, Vault will remain permanently sealed!" 
instead of 
"keys"

**To Reproduce**
1. Run `vault operator init`
2. See error

**Expected behavior**
The output to print the correct grammar (keys). 

**Environment:**
* Vault Server Version (retrieve with `vault status`): 
```
$ vault status
Key                Value
---                -----
Seal Type          shamir
Initialized        true
Sealed             true
Total Shares       5
Threshold          3
Unseal Progress    0/3
Unseal Nonce       n/a
Version            1.7.2
Storage Type       consul
HA Enabled         true
```
* Vault CLI Version (retrieve with `vault version`):
```
$ vault version
Vault v1.7.2 (db0e4245d5119b5929e611ea4d9bf66e47f3f208)
```
* Server Operating System/Architecture: 
```
Centos Linux 8
```

**Comment by balent** [comment]
Pull request fixed the bug. This issue can be closed.

**Comment by ncabatoff** [comment]
Agreed, this was fixed in #11792 .
